ml-agents/mlagents/trainers/trainer_util.py

```diff
@@ -190,3 +190,33 @@ def _load_config(fp: TextIO) -> Dict[str, Any]:
             "Error parsing yaml file. Please check for formatting errors. "
             "A tool such as http://www.yamllint.com/ can be helpful with this."
         ) from e
+
+
+def handle_existing_directories(
+    model_path: str, summary_path: str, resume: bool, force: bool
```

Comment:
note that `summary_path` isn't used yet but with consolidating all summaries into the same directory will be checked as well.